docs/guides/Checkout/orders-2/create-a-regular-order-from-an-existing-cart.md

```diff
@@ -26,8 +26,8 @@ To create an order via a specific shopping cart, you need to use the [Place orde
 Additionally, you need to send the request body containing the following information:
 
 - `referenceId`: ID of the orderForm corresponding to the cart in which the order will be created (`orderFormId`).
-- `savePersonalData`: this parameter should be set as "true" if the shopper’s data provided during checkout should be saved for future reference. Default value is "false".
-- `optinNewsLetter`: this parameter should be set as "true" if the shopper opted to receive the newsletter. Default value is "false".
+- `savePersonalData`: This parameter should be set as `true` if the shopper’s data provided during checkout should be saved for future reference. The default value is `true`.
+- `optinNewsLetter`: this parameter should be set as "true" if the shopper opted to receive the newsletter. Default value is `false`.
 - `value`: total value of the order without separating cents. For example, $24.99 is represented by `2499`.
 - `referenceValue`: reference value of the order for calculating interest (if applicable). It can be equal to the total value and does not separate cents. For example, $24.99 is represented by `2499`.
 - `interestValue`: interest rate to be used in case it applies.
```

Comment:
```suggestion
- `optinNewsLetter`: This parameter should be set as "true" if the shopper opted to receive the newsletter. Default value is `false`.
- `value`: Total value of the order without separating cents. For example, $24.99 is represented by `2499`.
- `referenceValue`: Reference value of the order for calculating interest (if applicable). It can be equal to the total value and does not separate cents. For example, $24.99 is represented by `2499`.
- `interestValue`: Interest rate to be used in case it applies.
```